platforms-serverless-vercel/vercel-with-redwood/api/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "@prisma/client": "4.13.0-dev.19",
     "@redwoodjs/api": "3.8.0",
-    "@redwoodjs/graphql-server": "3.8.0"
+    "@redwoodjs/graphql-server": "3.8.0",
+    "prisma": "4.13.0-dev.19"
```

Comment:
This additiona vs. the "resolutions" a few lines below?
One can go I guess?

Reply:
Yes, I'll clean that up